edgexfoundry/edgex-go
# fix: Consume Messaging Redis Subscribe fix for Request API

This fixes issue with Core Metadata timeout on the Device Validation requests sent to device services.

Also added more debug logging to help in future.







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run non-secure Edgex stack with device virtual and device rest
Verify core metadata logs show timeout errors
Verify device service logs show similar errors.
Build core metadata docker image locally from this branch
Stop/clean EdgeX stack. (make clean)
Edit compose file to use local dev image for core-metadata
Start stack using `make up`
Verify core metadata logs no longer has timeout errors
Verify device service logs no longer have errors.


## New Dependency Instructions (If applicable)
